docs/src/components/Palette/ColorContainer.tsx

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import convertHexToRgba from "@kiwicom/orbit-design-tokens/lib/convertHexToRgba.js";
+import { Stack } from "@kiwicom/orbit-components";
+
+import CopyButton from "./CopyButton";
+
+import { isLight, tokenString } from ".";
+
+interface BorderingProps {
+  order?: "first" | "middle" | "last" | "only";
+  full?: boolean;
+  left?: boolean;
+  right?: boolean;
+}
+
+export interface ColorValueShape {
+  colorValue: string;
+}
+
+const CopyWrapper = styled.div<ColorValueShape>`
+  ${({ colorValue, theme }) => css`
+    background: linear-gradient(90deg, transparent, ${colorValue} 27%);
+    position: absolute;
+    bottom: 20px;
+    right: ${theme.orbit.spaceLarge};
+    padding-left: ${theme.orbit.spaceXLarge};
+  `}
+`;
+
+interface ColorContainerWrapperProps extends ColorValueShape, BorderingProps {}
```

Comment:
typescript has type intersection, it's possible to pass to SC as `ColorValueShape & BorderingProps`

Reply:
I think this started because there had been props in the container that were unique to it. But as I simplified those out, I guess I just left the empty object. Should be intersectified now.